src/main/resources/static/external-merge-check-hook.soy

```diff
@@ -61,4 +61,26 @@
             ]
         ]/}
     {/call}
+	
+	{call aui.form.checkboxField}
+        {param legendContent: 'Cache Check Results' /}
+        {param fields: [
+            [
+                'id': 'cache_pr_results',
+                'labelText': 'Cache results of pull request check',
```

Comment:
Can you please also add `descriptionText` with something like:

```
Without this option specified executable will be executed whenever user will open pull request page.
```

Reply:
Added.